main.js

```diff
@@ -133,6 +151,35 @@ define(function (require, exports, module) {
         };
     }
     
+    /**
+     * Apply the whitespace colors.
+     * This does not overwrite styles already defined by a theme.
+     */
+    function _applyColors() {
+        // Compile the CSS from the template
+        var userColors = _preferences.get("colors"),
+            template   = _.template(stylesTemplate),
+            css = template({
```

Comment:
You could rearrange this a little bit by just passing in userColor and letting the css template file access the properties directly.  That way when new properties are added to userColors, you only need to change the css template file.  Also reduces the code you have right here in the JavaScript side.